src/applications/disability-benefits/526EZ/helpers.jsx

```diff
@@ -20,7 +20,8 @@ import { pick } from 'lodash';
 import { genderLabels } from '../../../platform/static-data/labels';
 
 import { DateWidget } from 'platform/forms-system/src/js/review/widgets';
-import { AddressViewField, capitalizeEachWord } from '../all-claims/utils';
+import AddressViewField from 'platform/forms-system/src/js/components/AddressViewField';
+import { capitalizeEachWord } from '../all-claims/utils';
```

Comment:
For what it's worth, this code `526 v1` was only supposed to be around for as long as there were unexpired saved forms for it. We might be at a place where the saved forms have all expired. It's probably worth looking into.

If we no longer have unexpired saved forms for v1, we can remove everything in `526EZ/`.

Reply:
I'll verify this with our team, and if true, I'll grab my sledgehammer! 🔨